sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/projectoverview/Project.groovy

```diff
@@ -0,0 +1,63 @@
+package life.qbic.business.projectoverview
+
+import life.qbic.datamodel.samples.Sample
+
+/**
+ * <h1><short description></h1>
+ *
+ * <p><detailed description></p>
+ *
+ * @since 1.0.0
```

Comment:
I think having a description similar to the Offer entity description in the offer-manager application makes sense here.